pkg/java/pom/parse.go

```diff
@@ -247,8 +247,13 @@ func (p *parser) analyze(pom *pom, exclusions map[string]struct{}) (analysisResu
 	depManagement := p.dependencyManagement(pom.content.DependencyManagement.Dependencies.Dependency, props)
 	depManagement = p.mergeDependencyManagement(parent.dependencyManagement, depManagement)
 
+	// depManagement deps from root file have higher priority
+	for k, v := range rootDependencyManagement {
+		depManagement[k] = v
+	}
+
```

Comment:
Can we put it into `mergeDependencyManagement`?

```
func (p parser) mergeDependencyManagements(depManagements ...map[string]pomDependency) map[string]pomDependency {
	new := map[string]pomDependency{}
	// The preceding argument takes precedence.
	for i := len(depManagements) - 1; i >= 0; i-- {
		for key, value := range depManagements[i] {
			new[key] = value
		}
	}
	return new
}
```

Reply:
Good idea!
Done